### ISSUE TITLE
Update PHP

### ISSUE THREAD

**Issue by JesiBuell79** [cul-it/uls#1180]
to latest version that works both for Upstream and unit sites

**Comment by sarah-cul** [comment]
[siteimprove plugin ](https://wordpress.org/plugins/siteimprove/ )does not work with php 8.1
we are using an older version and cannot download a newer version.
some of the unit sites do not have this activated. the annex site does and it should be deactivated for now until a safer version is available. 

NOTE: siteimprove can monitor the sites without this plugin, i don't know the benefit of using the plugin other than to get the information right from the wpadmin? this might be something the content owners requested access to. 

<img width="452" alt="image" src="https://github.com/cul-it/uls/assets/144364984/07423685-d27a-420f-8157-070b88fbbec9">

**Comment by sarah-cul** [comment]
For our records.. this is the feedback from siteimprove: 

```
Hi Sarah,

The plugin is currently unavailable through WordPress as we make some updates to security-related items for it, but we do have it available locally if needed. I've attached the current build's zip file to this email. We have no plans to discontinue the plugin, and should have it available again on WordPress soon.

Let me know if you have any additional questions!

Chris Knutson
Pronouns: He/Him/His
Senior Customer Support Engineer
Siteimprove, Inc.

Ticket ID: 2940144
```

**Comment by sarah-cul** [comment]
Upstream v2.6.5 included the update to php 8.1
